sdk/appcenter-crashes/src/test/java/com/microsoft/appcenter/crashes/CrashesTest.java

```diff
@@ -1159,12 +1162,21 @@ public ManagedErrorLog answer(InvocationOnMock invocation) {
         verify(mockChannel, never()).enqueue(any(ManagedErrorLog.class), eq(crashes.getGroupName()), anyInt());
     }
 
-    private ManagedErrorLog testNativeCrashLog(long appStartTime, long crashTime, boolean correlateSession) throws Exception {
+    private ManagedErrorLog testNativeCrashLog(long appStartTime, long crashTime, boolean correlateSession, boolean hasDeviceInfo) throws Exception {
 
         /* Setup mock for a crash in disk. */
         File minidumpFile = mock(File.class);
         when(minidumpFile.getName()).thenReturn("mockFile");
+        when(minidumpFile.isDirectory()).thenReturn(false);
         when(minidumpFile.lastModified()).thenReturn(crashTime);
+
+        /* Mock sub-folder. */
+        File minidumpSubfolder = mock(File.class);
```

Comment:
Same

Reply:
I can't achieve 100% coverage without it for the moment.